packages/testing/addon/mirage-graphql/schema.graphql

```diff
@@ -1,3 +1,13 @@
+"""
+Exposes a URL that specifies the behaviour of this scalar.
+"""
+directive @specifiedBy(
```

Comment:
How does this work, how is it used in the end?

Reply:
No idea. @winged might be able to help you.